database/rrd.h

```diff
@@ -738,6 +738,9 @@ struct rrdhost {
     RRDCALCTEMPLATE *templates;
     RRDCALCTEMPLATE *alarms_template_with_foreach;
 
+    //Index used with health
+    avl_tree_lock alarms_idx_health_name;
+    struct rrdcalc_rrdset_alarm *alarm_set;
```

Comment:
I don't see `alarm_set` being used anywhere in the code, other than iterating for `alarm_index_link()` and `alarm_index_unlink_and_free()` but I don't see how they're doing something with it.

Reply:
This is also used inside `rrdhost` in the same functions that I described before.

Reply:
But `alarm_set` doesn't do anything. `avl_remove_lock()` does all the work.

Reply:
It was removed with the last commit.